R/zchunk_L141.hfc_R_S_T_Y.R

```diff
@@ -1,42 +1,39 @@
 #' module_emissions_L141.hfc_R_S_T_Y
 #'
-#' Briefly describe what this chunk does.
+#' Calculated HFC emissions from EDGAR, by residential and commercial cooling shares,
+#' adjusted to match Guus Velders HFC inventory
```

Comment:
That's what it seems like. It's the "citation" on another inventory we use, but I can't find the actual citation. Was going to open an Issue.